myhdl/_block.py

```diff
@@ -275,10 +275,10 @@ def convert(self, hdl='Verilog', **kwargs):
             setattr(converter, k, v)
         return converter(self)
 
-    def config_sim(self, trace=False):
+    def config_sim(self, trace=False, timescale='1ns'):
```

Comment:
This is ok for now but we really need a solution for all of the trace and simulation options.  This probably needs to be extended - made generic to pass a bunch of args (kwards) and then the function needs to set the sim and trace parameters.

Reply:
This is already done in PR #201.